setup.py

```diff
@@ -22,13 +21,13 @@ def main():
     test_reqs = read_requirements('requirements_test.txt')
 
     setup(
-        name='pgcontents',
+        name='hybrid-content-manager',
         version='0.6',
-        description="A Postgres-backed ContentsManager for IPython/Jupyter.",
+        description="Hybrid Content Manager",
         long_description=long_description,
-        author="Scott Sanderson",
-        author_email="ssanderson@quantopian.com",
-        packages=find_packages(include='pgcontents.*'),
+        author="Viaduct AI",
+        author_email="engineering@viaduct.ai",
+        packages=find_packages(include='hybrid_content_manager.*'),
         license='Apache 2.0',
         include_package_data=True,
         zip_safe=False,
```

Comment:
Check to see if we still need this https://setuptools.readthedocs.io/en/latest/setuptools.html#setting-the-zip-safe-flag